generalInfoPage/general.html

```diff
@@ -21,7 +21,32 @@
         </nav>
     </header>
 
+    <section class="info-section">
+      <img class="country-img" src="/img/Turkey.png" alt="">
```

Comment:
name of the image file, not camelCase as it should

Reply:
done

Reply:
You change the file name but you didn't change it in the src line

Reply:
done